modules/Microsoft.Web/sites/deploy.bicep

```diff
@@ -218,6 +266,17 @@ resource app 'Microsoft.Web/sites@2021-03-01' = {
     keyVaultReferenceIdentity: !empty(keyVaultAccessIdentityResourceId) ? keyVaultAccessIdentityResourceId : null
     virtualNetworkSubnetId: !empty(virtualNetworkSubnetId) ? virtualNetworkSubnetId : any(null)
     siteConfig: siteConfig
+    clientCertEnabled: clientCertEnabled
```

Comment:
I see there a lot of common parameters for parent and child slots. Would it make sense to research if there are any constraints in their values? For instance, is it possible to have clientCertEnabled disabled for the parent and enabled for the slot? I'm not raising this to block the PR, but to double check if we should create a separate issue on that.

Reply:
As I understood the previous PRs, they are pretty much the same.

Reply:
To your question - no idea unfortunately if each slot is considered an independent app (which I think is the ideal) I would expect it works